### ISSUE TITLE
New structure of the repo

### ISSUE THREAD

**Issue by mrakitin** [Synchrotron-Radiation-Workshop/SRW#7]
```
SRW
	/cpp
		/gcc
		/vc
		/py
		/xcode
		/src
			/clients
				/c
				/igor
				/python

				/ext_lib (?)

			/core
			/ext
				/auxparse
				/genesis
				/genmath
			/lib
	/env
		/srw_python
		/srw_igor

	/ext_lib (move to cpp/src/ ?)

	/literature
```

**Comment by hhslepicka** [comment]
- Why `srw_dev` and not just `srw`?
- What does `env` mean in that context? Would it be better a more explicit name such as wrappers or interfaces?

**Comment by ChristopherMayes** [comment]
I agree, why not just `srw`?

**Comment by mrakitin** [comment]
It is meant to be the root of the repo, so it'll be the same repo name as it is now - `SRW`.

**Comment by mrakitin** [comment]
Closed via #9.
